frontend/taipy-gui/src/components/Taipy/MenuCtl.tsx

```diff
@@ -35,6 +42,7 @@ const MenuCtl = (props: MenuCtlProps) => {
         defaultLov = "",
         width = "15vw",
         width_Mobile_ = "85vw",
+        defaultSelectedIds = "",
```

Comment:
unnecessary, just handle value & defaultValue

Reply:
but if we use value & defaultValue to handle the selected_ids/items then we need to set it as default value right ?

Reply:
I suppose you mean default property ? 
No it is not necessary, just need to change from using updateVarName togetUpdateVar(props.updateVars, "<property name>")